src/App.js

```diff
@@ -8,7 +8,12 @@ import ContactScreen from './components/Contact';
 import NotificationsScreen from './components/Notifications';
 import CartScreen from './components/Cart';
 import { Checkout, CheckoutSuccess } from './components/Checkout';
-import AddFarmScreen from './components/AddFarm';
+import ProfileScreen from './components/Profile';
+import SignInScreen from './components/SignIn';
+import SignUpScreen from './components/SignUp';
+import {
+  LandingScreen, ForBuyersScreen, ForSellersScreen, AboutScreen,
+} from './components/PublicView';
 
```

Comment:
<img width="181" alt="Screen Shot 2021-05-11 at 1 22 55 PM" src="https://user-images.githubusercontent.com/59102166/117879681-17be1280-b25c-11eb-8397-db618190189b.png">
Quick UI Fix, but can you include the word pallets beneath the quantity of each of the cart items? This was added pretty recently, so you couldn't have known this will you were developing!